### PR TITLE
Pre-release 1.1.0 [5581]

### PR DESCRIPTION
**Pull request by julianbermudez**
This pull request adapts the documentation to the new Release 1.1.0.

### CODE REVIEW

File: docs/agent.rst [hunk 1]
```diff
@@ -18,70 +18,92 @@ There are several configuration parameters which can be set at **compile time**
...
+
+    $ ./MicroXRCEAgent udp [OPTIONS]
+
+    Options:
+      -h,--help                               Print this help message and exit
+      -p,--port UINT REQUIRED                 Select the port
+      -m,--middleware TEXT in {ced,dds}=dds   Select the kind of Middleware
+      -r,--refs FILE                          Load a reference file
+      -v,--verbose UINT in {0,1,2,3,4,5,6}=4  Select log level from less to more verbose
+      -d,--discovery                          Active the Discovery server
```
Comment (BorjaOuterelo) [inline docs/agent.rst]:
```suggestion
      -d,--discovery                          Activate the Discovery server
```

File: docs/agent.rst [hunk 2]
```diff
@@ -18,70 +18,92 @@ There are several configuration parameters which can be set at **compile time**
...
+
+    $ ./MicroXRCEAgent tcp [OPTIONS]
+
+    Options:
+      -h,--help                               Print this help message and exit
+      -p,--port UINT REQUIRED                 Select the port
+      -m,--middleware TEXT in {ced,dds}=dds   Select the kind of Middleware
+      -r,--refs FILE                          Load a reference file
+      -v,--verbose UINT in {0,1,2,3,4,5,6}=4  Select log level from less to more verbose
+      -d,--discovery                          Active the Discovery server
```
Comment (BorjaOuterelo) [inline docs/agent.rst]:
```suggestion
      -d,--discovery                          Activate the Discovery server
```

File: docs/agent.rst [hunk 3]
```diff
@@ -18,70 +18,92 @@ There are several configuration parameters which can be set at **compile time**
...
+      --disport UINT=7400 Needs: --discovery  Select the port for the Discovery server
+      --p2p UINT                              Activate the P2P profile using the given port
+
+**Serial server** (only Linux) ::
+
+    $ ./MicroXRCEAgent serial
```
Comment (BorjaOuterelo) [inline docs/agent.rst]:
```suggestion
    $ ./MicroXRCEAgent serial [OPTIONS]
```

File: docs/agent.rst [hunk 4]
```diff
@@ -18,70 +18,92 @@ There are several configuration parameters which can be set at **compile time**
...
+
+    Options:
+      -h,--help                               Print this help message and exit
+      --dev FILE REQUIRED                     Select the serial device
+      -b,--baudrate TEXT=115200               Select the baudrate
+      -m,--middleware TEXT in {ced,dds}=dds   Select the kind of Middleware
+      -r,--refs FILE                          Load a reference file
+      -v,--verbose UINT in {0,1,2,3,4,5,6}=4  Select log level from less to more verbose
+      -d,--discovery                          Active the Discovery server
```
Comment (BorjaOuterelo) [inline docs/agent.rst]:
```suggestion
      -d,--discovery                          Activate the Discovery server
```

File: docs/agent.rst [hunk 5]
```diff
@@ -18,70 +18,92 @@ There are several configuration parameters which can be set at **compile time**
...
+      --disport UINT=7400 Needs: --discovery  Select the port for the Discovery server
+      --p2p UINT                              Activate the P2P profile using the given port
+
+**Pseudo-Serial server** (only Linux) ::
+
+    $ ./MicroXRCEAgent pseudo-serial
```
Comment (BorjaOuterelo) [inline docs/agent.rst]:
```suggestion
    $ ./MicroXRCEAgent pseudo-serial [OPTIONS]
```

File: docs/agent.rst [hunk 6]
```diff
@@ -18,70 +18,92 @@ There are several configuration parameters which can be set at **compile time**
...
+
+    Options:
+      -h,--help                               Print this help message and exit
+      --dev FILE REQUIRED                     Select the serial device
+      -b,--baudrate TEXT=115200               Select the baudrate
+      -m,--middleware TEXT in {ced,dds}=dds   Select the kind of Middleware
+      -r,--refs FILE                          Load a reference file
+      -v,--verbose UINT in {0,1,2,3,4,5,6}=4  Select log level from less to more verbose
+      -d,--discovery                          Active the Discovery server
```
Comment (BorjaOuterelo) [inline docs/agent.rst]:
```suggestion
      -d,--discovery                          Activate the Discovery server
```

File: docs/agent.rst [hunk 7]
```diff
@@ -18,70 +18,92 @@ There are several configuration parameters which can be set at **compile time**
...
+      --disport UINT=7400 Needs: --discovery  Select the port for the Discovery server
+      --p2p UINT                              Activate the P2P profile using the given port
+
+* ``-d,--discovery <discovery-port>`` select a port for the discovery server which has the port number ``<7400>`` by default.
```
Comment (BorjaOuterelo) [inline docs/agent.rst]:
```suggestion
```

File: docs/agent.rst [hunk 8]
```diff
@@ -18,70 +18,92 @@ There are several configuration parameters which can be set at **compile time**
...
+* ``-r,--refs <refs-file>``: load a reference file using the relative path to the working directory.
```
Comment (BorjaOuterelo) [inline docs/agent.rst]:
```suggestion
```

File: docs/agent.rst [hunk 9]
```diff
@@ -18,70 +18,92 @@ There are several configuration parameters which can be set at **compile time**
...
   This reference file shall be composed by a set of Fast RTPS profiles following the `XML syntax <https://eprosima-fast-rtps.readthedocs.io/en/latest/xmlprofiles.html>`_ described in Fast RTPS.
```
Comment (BorjaOuterelo) [inline docs/agent.rst]:
```suggestion
  The reference file shall be composed by a set of Fast RTPS profiles following the `XML syntax <https://eprosima-fast-rtps.readthedocs.io/en/latest/xmlprofiles.html>`_ described in Fast RTPS.
```

File: docs/notes.rst [hunk 10]
```diff
@@ -1,6 +1,122 @@
...
+        * New unitary tests.
+        * API documentation.
+        * Raspberry Pi support.
+    * Change
+        * Memory usage improvement.
+        * CMake approach.
+        * Discovery API.
+        * Examples usage.
+    * Fix
+        * Acknack reading.
```
Comment (BorjaOuterelo) [inline docs/notes.rst]:
```suggestion
        * Acknack reading.
        * User data bad alignment.
```

File: docs/agent.rst (hunk 1 above)
Comment (julianbermudez) [reply docs/agent.rst]: Fixed!

File: docs/agent.rst (hunk 2 above)
Comment (julianbermudez) [reply docs/agent.rst]: Fixed!

File: docs/agent.rst (hunk 4 above)
Comment (julianbermudez) [reply docs/agent.rst]: Fixed!

File: docs/agent.rst (hunk 5 above)
Comment (julianbermudez) [reply docs/agent.rst]: Fixed!

File: docs/agent.rst (hunk 6 above)
Comment (julianbermudez) [reply docs/agent.rst]: Fixed!

File: docs/agent.rst (hunk 7 above)
Comment (julianbermudez) [reply docs/agent.rst]: Removed!

File: docs/agent.rst (hunk 8 above)
Comment (julianbermudez) [reply docs/agent.rst]: Removed!

File: docs/agent.rst (hunk 9 above)
Comment (julianbermudez) [reply docs/agent.rst]: Fixed!

File: docs/notes.rst (hunk 10 above)
Comment (julianbermudez) [reply docs/notes.rst]: Added!

**Pull request merged**